spec-main/api-safe-storage-spec.ts

```diff
@@ -0,0 +1,24 @@
+import { expect } from 'chai';
+import { safeStorage } from 'electron/main';
+
+describe('safeStorage module', () => {
+  describe('SafeStorage.isEncryptionAvailable()', () => {
+    it('should return true when encryption key is available', () => {
+      expect(safeStorage.isEncryptionAvailable()).to.equal(true);
+    });
+  });
+
+  describe('SafeStorage.encryptString()', () => {
+    it('valid input should correctly encrypt string', () => {
+      const plaintext = 'plaintext';
+      expect(String(safeStorage.encryptString(plaintext))).to.equal('v10NT@�\u001f2�\u001f��2�j\u0011�L');
```

Comment:
This isn't going to work, the encryption key varies from machine to machine by design.  You should just assert that this returned a buffer 🤷